FirebaseAuth/Tests/Unit/AuthKeychainServicesTests.swift

```diff
@@ -12,180 +12,185 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-import Foundation
-import XCTest
+// TODO(ncooke3): Add documentation for manually configuring tests on macOS.
+#if !os(macOS) || (os(macOS) && FIREBASE_AUTH_TESTING_USE_MACOS_KEYCHAIN)
```

Comment:
Where is `FIREBASE_AUTH_TESTING_USE_MACOS_KEYCHAIN` defined?

Reply:
it's not defined anywhere but i have a todo to provide a workflow for locally testing against the real keychain on macos. setting this preprocessor to 1 will be part of the workflow.